### DISCUSSION TITLE
dRHOdT from pop-tool.eos() is derivative of rho with respect to potential or in-situ temperature?

### DISCUSSION THREAD

**Discussion by DapengLi17** [NCAR/pop-tools#70] [Q&A]
Hi Sir/Lady:
    I am analyzing CESM POP outputs and using pop-tools.eos() to compute dRHOdT. I read the doc that dRHOdT is the derivative of rho with respect to temperature. Is dRHOdT the derivative of rho with respect to potential or in-situ temperature? There seems to be a typo in the attribute (unit) of dRHOdS in line 108 of eos.py. I think the unit should be kg/m^3/psu instead of kg/m^3/degC.  
     Any replies are highly appreciated! 
Regards!
Dapeng

**Comment by andersy005** [comment]
Hi @DapengLi17, I'm moving this issue to the discussions page since the discussions page is suitable for Q&A.

**Comment by andersy005** [comment]
Cc @NCAR/pop-tools-dev

**Reply by DapengLi17** [reply]
Thanks so much, @andersy005

**Reply by andersy005** [reply]
Gently pinging @NCAR/pop-tools-dev again in case someone has an answer to @DapengLi17 question

**Reply by andersy005** [reply]
> Gently pinging @NCAR/pop-tools-dev again in case someone has an answer to @DapengLi17 question

@NCAR/pop-tools-dev, thoughts/comments?
